index.html

```diff
@@ -64,6 +64,51 @@ <h2 class="title-2">I'm a software developer</h2>
 
         <i class="fas fa-chevron-down"></i>
     </section>
+    <div class="modal-popup" id="modal-popup">
+      <div class="modal-popup-content">
+        <div class="content-popup">
+          <button type="button" class="close" id="close-popup">X</button>
+          <img src="images/Snapshoot_Portfolio.png" alt="project image" />
+          <div class="title-and-buttons">
+            <h3 class="popup-title">Keeping track of hundreds of<br>components</h3>
+            <div class="popup-buttons">
+              <button type="button" class="popup-button">
+                See live&nbsp;&nbsp;&nbsp;<i class="fas fa-external-link-alt"></i>
+              </button>
+              <button type="button" class="popup-button">
+                See source&nbsp;&nbsp;&nbsp;<i class="fab fa-github"></i>
+              </button>
+            </div>
+          </div>
+          <div class="popup-tags">
+            <p>Codekit</p>
+            <p>GitHub</p>
+            <p>Javascript</p>
+            <p>Bootstrap</p>
+            <p>Terminal</p>
+            <p>Codepen</p>
+          </div>
+          <p class="popup-paragraph">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it 1960s with the releaLorem Ipsum is simply dummy
+            text of the printing and typesetting industry. Lorem Ipsum has been
+            the industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a galley of type and scrambled it 1960s with
+            the relea
+          </p>
+          <div class="popup-buttons-mobile">
+            <button type="button" class="popup-button-mobile">
+              See live&nbsp;&nbsp;&nbsp;<i class="fas fa-external-link-alt"></i>
+            </button>
+            <button type="button" class="popup-button-mobile">
+              See source&nbsp;&nbsp;&nbsp;<i class="fab fa-github"></i>
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
     <div>
         <h2 class="title-card-global">
             My Recent Works
```

Comment:
- [ ] It would be better to fix the alignment of `My Recent Works` and the `I'm always ...` title form too. It will help you to have a great project to show or use as the main portfolio.

![align-title-2](https://user-images.githubusercontent.com/79876959/142484687-187924fb-1e19-41aa-a318-a4474f0940e2.PNG)
![align-title-1](https://user-images.githubusercontent.com/79876959/142484701-2acdd4e8-77eb-4caa-b688-a1642fc1aab2.PNG)